string frame_id
string type
string color
float64 angle_t
float64 angle_b
float64 angle_l
float64 angle_r
